kythe/go/extractors/validation/validation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"strings"
 
 	"kythe.io/kythe/go/platform/kzip"
+	apb "kythe.io/kythe/proto/analysis_go_proto"
```

Comment:
Conventionally we group protobuf imports separately below the others.

Reply:
Done